runtime/src/main/scala/tailcall/runtime/service/HttpCache.scala

```diff
@@ -0,0 +1,56 @@
+package tailcall.runtime.service
+import tailcall.runtime.http.{HttpClient, Request}
+import zio.cache.{Cache, CacheStats, Lookup}
+import zio.http.Response
+import zio.{Duration, Exit, Task, ZIO, ZLayer}
+
+import java.text.SimpleDateFormat
+import java.time.Instant
+trait HttpCache  {
```

Comment:
This is basically an HttpClient. We can use it as `HttpClient.cached` to create a new layer of HttpClient which as caching capabilities.

Reply:
All services should be package private.